common/djangoapps/util/tests/test_password_policy_validators.py

```diff
@@ -10,13 +9,15 @@
 from django.core.exceptions import ValidationError
 from django.test.utils import override_settings
 
+from openedx.core.djangoapps.user_api.config.waffle import PASSWORD_UNICODE_NORMALIZE_FLAG
+from openedx.core.djangolib.testing.utils import CacheIsolationTestCase
 from util.password_policy_validators import (
     create_validator_config, validate_password, password_validators_instruction_texts,
 )
 
 
 @ddt
-class PasswordPolicyValidatorsTestCase(unittest.TestCase):
+class PasswordPolicyValidatorsTestCase(CacheIsolationTestCase):
```

Comment:
Just curious - why this change?

Reply:
Ran into a database failure trying to access the waffle flag. Super annoying.